### PR TITLE
Add new prow job for elcarro-oracle-operator

### PR DESCRIPTION
**Pull request by kchernyshev**

### CODE REVIEW

**Comment by kchernyshev** [comment]
FYI @aa3210 @kurt-google @yian-zhou

File: prow/prowjobs/GoogleCloudPlatform/elcarro-oracle-operator/elcarro-oracle-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
+        command:
+        - make
+        args:
+        - -C
+        - oracle
+        - operator-checks
+        securityContext:
```
Comment (kurt-google) [inline prow/prowjobs/GoogleCloudPlatform/elcarro-oracle-operator/elcarro-oracle-operator.yaml:16]: We can drop the securitycontext now, this was for KIND (docker in docker) tests.

**Review by kurt-google — APPROVED** [review]

File: prow/prowjobs/GoogleCloudPlatform/elcarro-oracle-operator/elcarro-oracle-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
+        command:
+        - make
+        args:
+        - -C
+        - oracle
+        - operator-checks
```
Comment (yian-zhou) [inline prow/prowjobs/GoogleCloudPlatform/elcarro-oracle-operator/elcarro-oracle-operator.yaml:15]: Shall we use operator-oss-presubmit instead?

**Review by yian-zhou — APPROVED** [review]

**Comment by kchernyshev** [comment]
Made minor updates according to Yian and Kurt

**Review by yian-zhou — APPROVED** [review]
Thanks Konstantin!

**Review by e-blackwelder — APPROVED** [review]

**Review by e-blackwelder — APPROVED** [review]

**Pull request merged**